docs/program/apis/cloud.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Accessing the Cloud with Viam's Client SDKs"
```

Comment:
What is accessing the cloud?  Maybe instead consider:
```suggestion
title: "Managing your fleet with Viam's cloud API"
```

Reply:
Good point!
```suggestion
title: "Managing Your Fleet with Viam's Cloud API"
```